fhir-bucket/src/main/java/com/ibm/fhir/bucket/client/FhirClientUtil.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.bucket.client;
+
+import java.io.ByteArrayOutputStream;
+import java.nio.charset.StandardCharsets;
+
+import com.ibm.fhir.model.format.Format;
+import com.ibm.fhir.model.generator.FHIRGenerator;
+import com.ibm.fhir.model.generator.exception.FHIRGeneratorException;
+import com.ibm.fhir.model.resource.Resource;
+
+/**
+ * Utilities for working with the FHIR client
+ */
+public class FhirClientUtil {
+    
+    
+    /**
+     * Render the resource as a string
+     * @param resource
+     * @return
+     */
+    public static String resourceToString(Resource resource) {
+        ByteArrayOutputStream os = new ByteArrayOutputStream(4096);
+        try {
+            FHIRGenerator.generator(Format.JSON, false).generate(resource, os);
+            return new String(os.toByteArray(), StandardCharsets.UTF_8);
+        } catch (FHIRGeneratorException e) {
+            throw new IllegalStateException(e);
+        }
+    }
```

Comment:
I'm almost certain there is an existing method for this in a lower level fhir-model - we should push this down there in that package.

Reply:
there probably should be, but this pattern is used all over

Reply:
This is not a comment on your code, more on the utility of this method.